x/params/client/cli/query.go

```diff
@@ -38,26 +39,15 @@ func NewQuerySubspaceParamsCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
+			queryClient := proposal.NewQueryClient(clientCtx)
 
-			params := types.NewQuerySubspaceParams(args[0], args[1])
-			route := fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryParams)
-
-			bz, err := clientCtx.JSONMarshaler.MarshalJSON(params)
-			if err != nil {
-				return fmt.Errorf("failed to marshal params: %w", err)
-			}
-
-			bz, _, err = clientCtx.QueryWithData(route, bz)
+			params := proposal.NewQueryParametersRequest(args[0], args[1])
+			res, err := queryClient.Parameters(context.Background(), params)
```

Comment:
Tests fails as soon as I use the queryClient here, not sure why.

```
    --- FAIL: TestIntegrationTestSuite/TestNewQuerySubspaceParamsCmd (0.00s)
        --- FAIL: TestIntegrationTestSuite/TestNewQuerySubspaceParamsCmd/default_output (0.00s)
            cli_test.go:85: 
                	Error Trace:	cli_test.go:85
                	            				suite.go:77
                	Error:      	Received unexpected error:
                	            	unknown query path: unknown request
```

Reply:
This means it didn't route/match correctly against the querier (gRPC?).

Reply:
I'm not familiar with the gRPC structure, but I would start debugging at the routing/matching level. Maybe this implementation is missing or invalid somehow?